### PR TITLE
Making the getting started docs more clear around environment variabes and their usage

### PR DESCRIPTION
**Pull request by ashahba**
Fixes #https://github.com/kubeflow/kubeflow/issues/1711

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/277)

### CODE REVIEW

**Comment by ashahba** [comment]
Since `KFAPP` environment variable is actually the `name` of a directory rather than it's `path`, I tried emphasizing the `name` part by surrounding it with a pair of back-ticks.
I hope that helps removing some of the previous ambiguity in the docs specially `getting-started` page.

**Comment by ashahba** [comment]
@richardsliu Please let me know if you want any more changes on this.

File: content/docs/started/getting-started-minikube.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ The following steps will deploy Kubeflow components and start them on the Miniku
     ${KUBEFLOW_SRC}/scripts/kfctl.sh generate all
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/started/getting-started-minikube.md]: Oh wait - see the [comment](https://github.com/kubeflow/website/pull/273/files#r229449690) from @jlewi on PR 273.

**Comment by sarahmaddox** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**